test/models/retrieveModels.m

```diff
@@ -21,35 +38,55 @@
     'Acinetobacter_calcoaceticus_PHEA_2.mat', 'https://webdav-r3lab.uni.lu/public/msp/AGORA/mat/Acinetobacter_calcoaceticus_PHEA_2.mat';
     };
 
+% download all models
 for i = 1:length(modelArr)
     if exist([MODELDIR, filesep, modelArr{i,1}], 'file') ~= 2
         urlwrite(modelArr{i,2}, modelArr{i, 1});
-        fprintf('Model %s saved.\n', modelArr{i, 1});
+        if printLevel > 0
+            fprintf(' + Downloaded:      %s\n', modelArr{i, 1});
+        end
     else
-        fprintf('%s %s\n', modelArr{i, 1}, ' already exists.');
+        if printLevel > 0
+            fprintf(' > Already exists:  %s\n', modelArr{i, 1});
+        end
     end
 end
 
-% Ec_iAF1260_flux1.xml
+% download Ec_iAF1260_flux1.xml
 if exist('Ec_iAF1260_flux1.xml', 'file') ~= 2
     urlwrite('http://systemsbiology.ucsd.edu/sites/default/files/Attachments/Images/InSilicoOrganisms/Ecoli/Ecoli_SBML/msb4100155-s6.zip', 'msb4100155-s6.zip');
     system('unzip -qq msb4100155-s6.zip');
     system('rm Ec_iAF1260_flux2.txt');
```

Comment:
rm and mv along with unzip are all unix commands. This will likely Fail on a Windows system.